group-coordinator/src/main/java/org/apache/kafka/coordinator/group/metrics/GroupCoordinatorRuntimeMetrics.java

```diff
@@ -120,19 +121,14 @@ public GroupCoordinatorRuntimeMetrics(Metrics metrics) {
                 "The average time it took to load the partitions in the last 30 sec."
             ), new Avg());
 
-        this.threadIdleRatioSensor = metrics.sensor("ThreadIdleRatio");
-        this.threadIdleRatioSensor.add(
-            metrics.metricName(
-                "thread-idle-ratio-min",
+        this.threadIdleSensor = metrics.sensor("ThreadIdleRatio");
+        this.threadIdleSensor.add(new Meter(TimeUnit.MILLISECONDS,
+            metrics.metricName("thread-idle-ratio",
```

Comment:
If I understand it correctly, we can an average out of this one. Should we keep the `-avg` suffix here?